change/@fluentui-react-charting-33f77694-92d5-460d-b4e5-d9cec93f81e2.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Breaking change fixed when chart data is empty",
```

@Raghurk should review this overall, but please note that in open source, the phrase "breaking change" typically means you're making a backwards-incompatible API change or a major behavior change. From briefly looking over your code, it appears to be just a bug fix.
```suggestion
  "comment": "Stacked Bar and Multi Stacked Bar chart: fixed when chart data is empty",
```